corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -118,6 +123,41 @@ def process_change(self, change):
     def fire_change_processed_event(self, change, context):
         pass
 
+    def _record_checkpoint_in_datadog(self):
+        for topic, value in self.get_last_checkpoint_sequence().iteritems():
+            datadog_gauge('commcare.change_feed.topics'.format(topic), points=value, tags={
+                'pillow_name': self.get_name(),
+                'topic': topic,
+            })
+            datadog_counter('commcare.change_feed.topics.counter'.format(topic), tags={
+                'pillow_name': self.get_name(),
+                'topic': topic,
+            })
```

Comment:
yeah so we can see the rate at which we create checkpoints. maybe not that useful